languages/elixir/exercises/concept/multiple-clause-functions/.docs/introduction.md

```diff
@@ -0,0 +1,58 @@
+Elixir allows programmers to write code that follows closely to the **Open-Close Principle**. That is, software should be open for extension, but closed for modification. This is an important principle, because as your software grows in size and complexity, changes to that software are more difficult to make.
+
+## Multiple Function Clauses
+
+Elixir facilitates following this principle by allowing functions to have multiple clauses, so instead of sprawling and hard-coded control-logic, pointed functions can be written to add/remove behavior easily.
+
+Suppose we have a function `number/1` to test whether a number is a favorite.
+
+```elixir
+def number(n) do
+  "Awesome, that's my favorite"
+end
+```
+
+Now suppose we have been asked to only have the number `7` be the favorite, a person might write:
+
+```elixir
+def number(n) do
+  cond do
+    number == 7 -> "Awesome, that's my favorite"
```

Comment:
Erik keeps saying that the introduction.md should only present new syntax, but the first two code examples here don't do that. Maybe just get rid of them and start straight with a multiple-clause function example, without showing alternatives like `cond`?